README.rst

```diff
@@ -41,6 +41,29 @@ project's virtual environment.
 
 .. _virtual environment: https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments
 
+
+Cross-environment usage of ``requirements.in``/``requirements.txt`` and ``pip-compile``
```

Comment:
Maybe this should go to the bottom of the file instead (just before "Note about pip")? Let's not confuse readers by this advanced stuff until they actually know the basics?

Reply:
Agreed, I'll move that down